spyder/plugins/application/widgets/status.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Spyder updates.
+"""
+
+# Standard library imports
+import logging
+
```

Comment:
I missed suggesting an import in my last review @jsbautista 😅 
```suggestion
import os

```